index.html

```diff
@@ -34,233 +38,314 @@ <h1 class="name">
 					Start collaboration
 				</button>
 			</div>
+			<div class="image_dots"></div>
 			<div class="bottom_geometry"></div>
 		</section>
 
 		<section class="project_section">
 			<h2 class="projects_label">
 				Projects
 			</h2>
-			<div class="project_card">
-				<div class="project_image"></div>
+			<div class="image_dot_1"></div>
+			<div class="project_card_macro">
 				<div class="project_details_card">
 					<h2 class="project_name">
 						Bavon's Portfolio
 					</h2>
 					<div class="project_tecs_container">
-						<ul class="ul">
-							<li class="li">
-								<div class="tech_card">
-									<p class="tech_text">
-										HTML/CSS
-									</p>
-								</div>
-							</li>
-							<li class="li">
-								<div class="tech_card">
-									<p class="tech_text">
-										Ruby on Rails
-									</p>
-								</div>
-							</li>
-							<li class="li">
-								<div class="tech_card">
-									<p class="tech_text">
-										JavaScript
-									</p>
-								</div>
-							</li>
-						</ul>
+							<ul class="ul">
+								<li class="li">
+									<div class="tech_card">
+										<p class="tech_text">
+											HTML/CSS
+										</p>
+									</div>
+								</li>
+								<li class="li">
+									<div class="tech_card">
+										<p class="tech_text">
+											Ruby on Rails
+										</p>
+									</div>
+								</li>
+								<li class="li">
+									<div class="tech_card">
+										<p class="tech_text">
+											JavaScript
+										</p>
+									</div>
+								</li>
+							</ul>
 					</div>
 					<button class="see_project">
 						See this project
 						<i class="arrow_forward"></i>
 					</button>
 				</div>
 			</div>
-			<div class="project_card">
+			<div class="project_section_images">
```

Comment:
- [ ] Some of these images are not aligned like in the Figma:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/149367570-2e975d9c-159a-476b-8fa1-8cb732ded09e.png) | ![image](https://user-images.githubusercontent.com/57421823/149367728-9c796496-5083-4af1-9253-5d1410bdda91.png)

Please kindly correct them, so they look like the design 💯